intake/source/cache.py

```diff
@@ -106,8 +121,14 @@ def load(self, urlpath):
             cache_path = file_out.path
             cache_paths.append(cache_path)
 
+            # If `_munge_path` did not find a match we want to avoid writing to the urlpath.
+            if cache_path == urlpath:
+                continue
+
             if not os.path.isfile(cache_path):
-                logger.info("Caching file {} from urlpath {}".format(file_in.path, urlpath))
+                logger.info("Caching file: {}".format(file_in.path))
```

Comment:
Change these to debug and add docs for how to set logging level.

Reply:
👍